src/Microsoft.OData.Core/JsonLight/ODataJsonLightPropertySerializer.cs

```diff
@@ -143,89 +143,98 @@ internal void WriteTopLevelProperty(ODataProperty property)
             IDuplicatePropertyNameChecker duplicatePropertyNameChecker,
             ODataResourceMetadataBuilder metadataBuilder)
         {
-            this.WritePropertyInfo(property, owningType, isTopLevel, duplicatePropertyNameChecker, metadataBuilder);
-
-            ODataValue value = property.ODataValue;
-
-            // handle ODataUntypedValue
-            ODataUntypedValue untypedValue = value as ODataUntypedValue;
-            if (untypedValue != null)
+#if NETSTANDARD2_0 || NETCOREAPP3_1_OR_GREATER

```

Comment:
why didn't we use NETSTANDARD2_0_OR_GREATER

Reply:
@ElizabethOkerio Because even in the existing code, e.g. _JsonReader.cs_, we use **`NESTANDARD2_0`** which the library targets specifically.

![image](https://user-images.githubusercontent.com/28291332/227154430-10fa6093-453d-4f8c-9d84-a18e1e0f5666.png)